.well-known/apple-app-site-association.json

```diff
@@ -0,0 +1,19 @@
+{
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "3YE4W86L3G.Tavultesoft.Keyman",
+        "paths": ["/keyboards/install/*"],
+
```

Comment:
```suggestion
}, {
```

Reply:
All examples I'm seeing online put the two definition styles in separate objects within the array, not within the same object.

We might should favor the top one and drop the bottom; the bottom is iOS 13.0+ friendly, whereas the top one supports older versions too.  Hopefully they can coexist, though.